webhook.go

```diff
@@ -4,26 +4,29 @@ import (
 	"time"
 )
 
+// WebhookConfig is a Webhook substructure with data related to event delivery.
+type WebhookConfig struct {
```

Comment:
Not a huge fan of this name, but not sure what a better name could be.  `DeliveryConfig`? `EventRequestConfig`? But the field is called config and we can't change that so also fine with leaving as is. 🤷

Reply:
I like `DeliveryConfig` 🚀